src/Riok.Mapperly.Abstractions/EnumNamingStrategy.cs

```diff
@@ -0,0 +1,22 @@
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// Defines the strategy to use when mapping an enum from/to string.
+/// </summary>
+public enum EnumNamingStrategy
+{
+    /// <summary>
+    /// Matches enum values using PascalCase.
+    /// </summary>
+    PascalCase,
+
+    /// <summary>
+    /// Matches enum values using snake_case.
+    /// </summary>
+    SnakeCase,
```

Comment:
Maybe also add camel case? System.Text.Json also has support for upper and lower snake and kebab casing, not sure if we should add that too. WDYT?

Reply:
Yes, I have added all the mentioned casing strategies.